lib/util/git/index.ts

```diff
@@ -266,6 +266,10 @@ export async function syncGit(): Promise<void> {
     try {
       await git.raw(['remote', 'set-url', 'origin', config.url]);
       await resetToBranch(await getDefaultBranch(git));
+      // istanbul ignore if
+      if (process.env.NODE_ENV !== 'test') {
+        await git.raw(['config', '--unset-all', 'remote.origin.fetch']);
```

Comment:
```suggestion
        await git.raw(['config', '--unset-all', 'remote.origin.fetch']);
```

Why not using `git.config` function? I'm not sure if this solves the issue. Could you reproduce it with local real run? It's a little complicated i think.

Reply:
![image](https://user-images.githubusercontent.com/6311784/120437212-20e46000-c380-11eb-88e4-f631eed7286c.png)
Seems it's not supported?

We use `git.raw` here too:
![image](https://user-images.githubusercontent.com/6311784/120437382-4c674a80-c380-11eb-8429-c9885e0cf673.png)

I can't reproduce this locally but it keeps happening in the app. I don't see much downside in trying this.

Reply:
It's `addConfig` function 🙈